types/account.go

```diff
@@ -1,5 +1,6 @@
 // Copyright Tharsis Labs Ltd.(Evmos)
 // SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
 package types
 
 import (
```

Comment:
If we're removing this account type and the `account.proto`, we should also remove the `EthAccountI` that uses the `EthAccount` type.
Also, we need to remove the `account.pb.go`

Reply:
Ok I can move it to a migration dir 👍

Reply:
the existing migration directories are only for in-store migrations when e.g. changing the params format though right? This is not applicable here and we're not upgrading the module version to v8 on this one. Do you think that's necessary? 🤔

Reply:
we're preforming a store migration in the accounts module. We're replacing all account types. 
Ideally this should be an account module store migration, but because is related to the evm, we can add it there.
The case that I'm thinking of is if a client has v16 and wants to update to v19, will have to manually add the logic in the upgrade handler, and if they miss this, then the application will probably break. If we include this as a migration within the evm module, the client will have this migration automatically. No need to add it explicitly in the upgrade handler

Reply:
In this way, the application module manager takes care of the corresponding migrations. This is the same approach than in all cosmos-sdk modules.